.github/workflows/gh-pages.yaml

```diff
@@ -11,6 +11,8 @@ jobs:
   deploy_preview:
     name: Deploy preview versions on pull requests
     runs-on: ubuntu-latest
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
We can still merge this part.

Reply:
yes, it's to avoid `The process '/usr/bin/git' failed with exit code 1`

Reply:
Ok, please push a new commit to remove the tag change above, then I'll merge this.

Reply:
updated on my side